src/include/module/module/api_ver.h

```diff
@@ -23,6 +23,13 @@
 #define SOF_MODULE_API_MIDDLE_VERSION	0
 #define SOF_MODULE_API_MINOR_VERSION	0
 
+/* Defines the size of the space reserved within ModuleHandle for SOF to store its private data.
+ * This size comes from the IADK header files and must match the IADK API version.
+ * Please do not modify this value!
+ */
+#define IADK_MODULE_PASS_BUFFER_SIZE	320
```

Comment:
this seems a rather strange placement for this macro - both the file and the place within it. It seems more suitable for an IADK specific header, like the one below